.github/workflows/lint.yml

```diff
@@ -17,8 +17,6 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
You can remove this whole `GenerateScript` job, it was only meant to make sure the generator script was updated.